safari/app-store-keywords.txt

```diff
@@ -0,0 +1 @@
+github,gh,git,safari,extension,browser,npm,python,issue,pull,request
```

Comment:
Do you know if there is a limit for keywords? I wonder whether or not we should list all tools and languages OctoLinker supports, like done in the description

Reply:
It's 100 characters [apparently](https://www.google.com/search?client=safari&rls=en&q=app+store+keyword+limit&ie=UTF-8&oe=UTF-8), the UI doesn't say anything